.github/workflows/linux.yml

```diff
@@ -146,10 +143,7 @@ jobs:
             test-image: "images:almalinux/9"
           - os: debian-bookworm
             package: mysql-community-8.0
-            test-image: "images:debian/12"
-          - os: ubuntu-focal
-            package: mysql-community-8.0
-            test-image: "images:ubuntu/20.04"
+            test-image: "images:debian/12""
```

Comment:
```suggestion
            test-image: "images:debian/12"
```

Reply:
fix [0d750af](https://github.com/mroonga/mroonga/pull/888/commits/0d750af2d8d38fb03c4e032206f329845df6ccf4): Thank you so much.